MANIFEST.in

```diff
@@ -6,4 +6,5 @@ include dash_cytoscape/metadata.json
 include dash_cytoscape/package.json
 include dash_cytoscape/utils
 include README.md
-include LICENSE
\ No newline at end of file
+include LICENSE
+include assets
```

Comment:
I'm inclined to say ONLY include `assets/contextmenu.css` in here, since the other files in `assets/` don't seem to be needed.

Reply:
Turns out base.css is also needed so I preferred adding the whole assets directory instead of specifying 2 individual files. What do you think?

Reply:
That makes sense but can we delete `fonts.css` and `override.css` entirely then, or move them to a different folder?

Reply:
@alexcjohnson Seems like these are not being used anywhere, do you think we could delete both of these?

Reply:
From what I can tell base.css is used and fonts.css and override.css is not.